packages/venia-ui/lib/components/CheckoutPage/checkoutPage.css

```diff
@@ -77,11 +77,17 @@
 }
 
 .summaryContainer {
-    grid-column: 2 / span 1;
-    grid-row: 1 / span 1;
-    height: 0;
-    position: sticky;
-    top: 5.5rem;
+    grid-column: 1 / span 1;
+}
+
+@media (min-width: 961px) {
+    .summaryContainer {
+        grid-column: 2 / span 1;
+        grid-row: 2 / span 3;
+        position: sticky;
+        top: 5.5rem;
+        height: min-content;
```

Comment:
The technique used here to position the summary container was intentional. Does this change attempt to resolve an edge case?

- this element shouldn't need to know how many rows are in the grid
- this element shouldn't affect row sizes in the grid

```stylus
.summaryContainer
  grid-row-: 1 / span 1 // place the element into only one row
  height: 0 // avoid contributing to implicit row height
```

Reply:
Yes, this is fixing edge case when summary can overlap footer when scrolling.